src/utils/tests/wbsNumSort.test.ts

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I just realized this test file is in the wrong location. Let's keep this test file in the same folder as the component where the function being tested is written.